api/src/mailbox.rs

```diff
@@ -422,6 +504,30 @@ impl Request for InvokeDpeReq {
     type Resp = InvokeDpeResp;
 }
 
+// EXTEND_PCR
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct ExtendPcrReq {
+    pub hdr: MailboxReqHeader,
+    pub pcr_idx: u32,
+    pub data: [u8; 48],
+}
+pub type ExtendPcrReqErr = ();
+impl ExtendPcrReq {
+    pub const DATA_MAX_SIZE: usize = 48;
```

Comment:
Given that data is always 48 bytes, no more no less, 'MAX' is somewhat meaningsless? Maybe drop this?